python/lsst/ip/isr/isrTask.py

```diff
@@ -574,6 +582,10 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
                 self.maskAndInterpNan(ccdExposure)
                 interpolationDone = True
 
+            if self.config.brighterFatterLevel == 'CCD':
```

Comment:
`CCD` -> `DETECTOR`

Reply:
Fixed here, and in the config option default and doc.